site/en/r2/tutorials/load_data/text.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>Let's do this</p><p><br></p><p>```</p><p>all_labeled_data = labeled_data_sets[0].concatenate(labeled_data_sets[1])</p><p><br></p><p>for index, val in enumerate(labeled_data_sets[2:]):</p><p>&nbsp;all_labeled_data = all_labeled_data.concatenate(val)</p><p>```</p><p><br></p><p>so that if you have more than 3 files, you won't have to manually concatenate it.</p><p><br></p><p>Also, shuffling here is a bad idea since you do .take() below. I would recommend doing the shuffle after .take() and .skip()</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>